template/android/app/build.gradle

```diff
@@ -159,19 +173,30 @@ android {
                 output.versionCodeOverride =
                         versionCodes.get(abi) * 1048576 + defaultConfig.versionCode
             }
+
         }
     }
+
+    packagingOptions {
+        pickFirst '**/armeabi-v7a/libc++_shared.so'
+        pickFirst '**/x86/libc++_shared.so'
+        pickFirst '**/arm64-v8a/libc++_shared.so'
+        pickFirst '**/x86_64/libc++_shared.so'
+        pickFirst '**/x86/libjsc.so'
+        pickFirst '**/armeabi-v7a/libjsc.so'
+    }
 }
 
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.facebook.react:react-native:+"  // From node_modules
 
-    // JSC from node_modules
-    if (useIntlJsc) {
-        implementation 'org.webkit:android-jsc-intl:+'
+    if (enableHermes) {
```

Comment:
what do you think about moving this to `react.gradle` so all apps don't have to manually do this? we'd need to be able to infer if a variant requires the release or debug aar. There's already `hermesFlagsRelease`/`hermesFlagsDebug ` so we could build on that concept.

Reply:
This change is really large, I prefer to land it and then hope the community will help improve it. Right now this is following what JSC is doing already.